src/index.html

```diff
@@ -17,11 +17,385 @@
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__content">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="card__content-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__content">
+          <span class="card__content-text">Price:</span>
+          <span class="card__content-price">$2,199</span>
+        </div>
+
+        <input
+          class="card__button"
+          type="button"
+          value="buy"
+          data-qa="card-hover"
+        />
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__content">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="card__content-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__content">
+          <span class="card__content-text">Price:</span>
+          <span class="card__content-price">$2,199</span>
+        </div>
+
+        <input
+          class="card__button"
+          type="button"
+          value="buy"
+        />
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```